app/src/main/java/com/hedvig/app/feature/embark/passages/previousinsurer/askforprice/AskForPriceInfoActivity.kt

```diff
@@ -66,6 +68,7 @@ class AskForPriceInfoActivity : BaseActivity() {
     }
 }
 
+@OptIn(ExperimentalAnimationApi::class)
 @Preview
 @Composable
 fun AskForPriceScreenPreview() {
```

Comment:
Probably the optin can be added only to the one directly using the experimental APIs right, this one doesn't seem to at its current state
Also having a preview before a non-preview composable is probably not a good idea?